src/CommentSold/Api/Services/AccountService.php

```diff
@@ -9,49 +9,52 @@
 use CommentSold\Api\Resources\Request\Account\CreateShopRequest;
 use CommentSold\Api\Resources\Request\Account\GetIframeUrlRequest;
 use CommentSold\Api\Resources\Request\Account\GetOauthUrlRequest;
+use CommentSold\Api\Resources\Response\Account\CreateShopResponse;
+use CommentSold\Api\Resources\Response\Account\GetIframeUrlResponse;
+use CommentSold\Api\Resources\Response\Account\GetOauthUrlResponse;
 use CommentSold\Api\ShopClient;
 
 class AccountService extends abstractService
 {
     /**
      * Create Shop
      */
-    public function createShop(CreateShopRequest $payload)
+    public function createShop(CreateShopRequest $payload): CreateShopResponse
```

Comment:
If there's nothing to return, you don't have to return anything (`: void`), as long as you have your http client (in the abstract class) configured to throw exceptions for non-200.